scripts/release/README.md

```diff
@@ -1,14 +1,16 @@
-# Helper for monitoring of release
-`pip install .`
-`monitor-release --help`
+# release-monitor
 
-Check current status of containers
-`monitor-release master containers`
-Check running sidecars:
-`monitor-release master sidecars`
 
-# Create .env file
-```
+Helper for monitoring of release. Here some one-liner examples:
+
+- Installation: `pip install .; monitor-release --help`
+- Check current status of containers in `master` deployment: `monitor-release master` containers.
+- Check running sidecars in `master` deployment: `monitor-release master containers`
+
+
+## Create .env file
```

Comment:
Is this not also via . makefile?

Reply:
No. this version still keeps this type of .env file. Once we test live we will remove these